src/gui/XournalView.cpp

```diff
@@ -257,14 +257,30 @@ bool XournalView::onKeyPressEvent(GdkEventKey* event)
 
 	if (event->keyval == GDK_KEY_KP_Left)
 	{
-		this->pageRelativeXY(-1, 0);
-		return true;
+		if (control->getSettings()->isPresentationMode() && control->getSettings()->getPresentationLeftRightEnabled())
+		{
+			control->getScrollHandler()->goToPreviousPage();
+			return true;
```

Comment:
Bring the `return true` outside of the two if statements since they're duplicated.

Reply:
Ah, right, I have fixed it, amended and force pushed, thanks!